DESCRIPTION

```diff
@@ -40,6 +40,7 @@ Imports:
 Suggests:
     broom (>= 0.7.10),
     car (>= 3.0-13),
+    checkmate,
```

Comment:
@averissimo Can I ask you how to update this correctly?

Reply:
Sure thing!

1. Version needs to be at least `checkmate (>= 2.1.0),`
    - Hard requirement as previous versions don't build anymore
3. Add repository hint `mllg/checkmate` to the `Config/Needs/verdepcheck` section below
    - I suggest as the first item on the line after `cran/car` to preserve the alphabetical order per section

```diff
diff --git a/DESCRIPTION b/DESCRIPTION
index ea5ae0bc..7719a39a 100644
--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -40,7 +40,7 @@ Imports:
 Suggests:
     broom (>= 0.7.10),
     car (>= 3.0-13),
-    checkmate,
+    checkmate (>= 2.1.0),
     dplyr (>= 1.0.5),
     flextable (>= 0.8.4),
     knitr (>= 1.42),
@@ -55,7 +55,7 @@ VignetteBuilder:
     knitr
 Config/Needs/verdepcheck: insightsengineering/formatters,
     tidyverse/magrittr, rstudio/htmltools, tidymodels/broom, cran/car,
-    tidyverse/dplyr, davidgohel/flextable, yihui/knitr,
+    mllg/checkmate, tidyverse/dplyr, davidgohel/flextable, yihui/knitr,
     davidgohel/officer, Merck/r2rtf, r-lib/testthat, tidyverse/tibble,
     tidyverse/tidyr, r-lib/xml2
 Encoding: UTF-8

```

Reply:
Super!! Thank you very much ;)